gephi/gephi-plugins
# HTTP Graph plugin update to 1.0.9 for Gephi 0.9.3

## New plugin or plugin update?


## What is the purpose of this plugin?
Graph relationships between web sites.  Since 2011, even. :D
  
## How to test your plugin in Gephi?

1. In Chrome: Install the [HTTP Graph Collector](https://chrome.google.com/webstore/detail/http-graph-collector/lkkdeokncfjlinldgikoabgknklnnkoe) extension.  It will POST the data in real time.  [Extension source code](https://github.com/phreakocious/HTTP-Graph-Collector)
2. In Gephi: New Project, File/Generate/HTTP Graph/Ok
3. Browse around in Chrome and you should see data start flowing in.

## Checklist before submission

* [ ] Did you write unit tests to test your plugin functionalities?

No unit tests..  I am my own unit tests?
